### PR TITLE
RNs-4.9.52 Added with Bug fix update

### PR DESCRIPTION
**Pull request by Wiharris**
**RNs-4.9.52**
Added with Bug fix update

**Version(s):**
* PR applies only to enterprise-4.9

**Issue:**
https://issues.redhat.com/browse/OSDOCS-4577

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/RNs-4.9.52/release_notes/ocp-4-9-release-notes.html#ocp-4-9-52

QE review:
- [ ] QE has approved this change.

**Additional information:**
z-stream is `shipped live`

### CODE REVIEW

**Comment by GroceryBoyJr** [comment]
Hey @Wiharris I can't find the term "IBM powerbuild architecture" anywhere, trying to determine how it might be capitalized. What I did find in the IBM Documentation is "IBM Power build"  What I did find is this: https://www.ibm.com/docs/en/rdfa-and-l/9.1.0?topic=data-best-practices-setting-power-build-configuration-options which describes IBM Power build options.

**Comment by GroceryBoyJr** [comment]
@Wiharris I can't see any QE approvals on this PR. Does it need any QE approvals?

**Comment by GroceryBoyJr** [comment]
@Wiharris peer review finished, only a couple nits.

**Review by GroceryBoyJr — COMMENTED** [review]
Only a couple small nits, PTAL.

**Comment by Wiharris** [comment]
> @Wiharris I can't see any QE approvals on this PR. Does it need any QE approvals?

Hi James! Usually we do not require QE approval for z-stream release notes.

**Comment by Wiharris** [comment]
> Hey @Wiharris I can't find the term "IBM powerbuild architecture" anywhere, trying to determine how it might be capitalized. What I did find in the IBM Documentation is "IBM Power build" What I did find is this: https://www.ibm.com/docs/en/rdfa-and-l/9.1.0?topic=data-best-practices-setting-power-build-configuration-options which describes IBM Power build options.

I can't find any references to the architecture in Power Build either. I did use the verbiage from https://github.com/openshift/openshift-docs/pull/52997/files as this is accross multiple z-stream updates.

**Comment by GroceryBoyJr** [comment]
@Wiharris Thank you sir, checking now.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -3295,3 +3295,22 @@ For more information, see xref:../authentication/bound-service-account-tokens.ad
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-52"]
+=== RHBA-2022:8485 - {product-title} 4.9.52 bug fix update
+
+Issued: 2022-11-22
+
+{product-title} release 4.9.52 is now available. 4.9.52 contains no IBM powerbuild architecture. IBM powerbuild architecture will be included in the next z-stream release. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:8485[RHBA-2022:8485] advisory. There are no RPM packages for this release.
```
Comment (GroceryBoyJr) [inline release_notes/ocp-4-9-release-notes.adoc:3304]: Recommend changing "IBM powerbuild architecture" to "IBM Power build options", per the IBM documentation.

**Review by GroceryBoyJr — COMMENTED** [review]
Only one little nit, otherwise LGTM. Thank you sir!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: Thanks, James! I have updated the PR to reflect the above changes. :)

**Review by bergerhoffer — COMMENTED** [review]
Thanks for updating the date, lgtm for merging now!

**Pull request merged**